build.sbt

```diff
@@ -514,7 +515,36 @@ lazy val scaladoc = configureAsSubproject(project)
   .settings(
     name := "scala-compiler-doc",
     description := "Scala Documentation Generator",
-    includeFilter in unmanagedResources in Compile := "*.html" | "*.css" | "*.gif" | "*.png" | "*.js" | "*.txt" | "*.svg" | "*.eot" | "*.woff" | "*.ttf"
+    includeFilter in unmanagedResources in Compile := "*.html" | "*.css" | "*.gif" | "*.png" | "*.js" | "*.txt" | "*.svg" | "*.eot" | "*.woff" | "*.ttf",
+    libraryDependencies += jqueryDep,
+    resourceGenerators in Compile += Def.task {
+      val resourcesFromJars = Map(
+        // resourceName -> jarName
+        "jquery.min.js" -> jqueryDep.name
```

Comment:
is there a way to do an integrity check on the jquery binary?

Reply:
Major CDNs offer integrity check via `integrity` attribute of script tag.
https://code.jquery.com/

I don't know how to check integrity on webjars not ordinal JARs.

Reply:
could it maybe be passed as an optional parameter to `extractResourceFromJar`?

Reply:
Added [SRI (Subresource Integrity)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) check  in https://github.com/scala/scala/pull/8179/commits/66ab1fe10ec5e333f55fc444341771e39905b20b

Reply:
nice! thanks!